### PR TITLE
feat(wallet_ui): add account details page

### PR DESCRIPTION
**Pull request by stringhandler**
Description
---
Adds an account details page.

### CODE REVIEW

File: applications/tari_dan_wallet_daemon/src/handlers/accounts.rs [hunk 1]
```diff
@@ -414,7 +418,8 @@ pub async fn handle_claim_burn(
         claim_proof,
         fee,
     } = req;
-
+    dbg!(claim_proof.clone());
```
Comment (Cifko) [inline applications/tari_dan_wallet_daemon/src/handlers/accounts.rs]: remove?

File: applications/tari_dan_wallet_daemon/src/jrpc_server.rs [hunk 2]
```diff
@@ -119,13 +120,27 @@ where
...
+                e
+            })?,
+        )
         .await
         .map_err(|e| resolve_handler_error(answer_id, &e))?;
     Ok(JsonRpcResponse::success(answer_id, resp))
 }
 
 fn resolve_handler_error(answer_id: i64, e: &HandlerError) -> JsonRpcResponse {
+    dbg!("here2");
```
Comment (Cifko) [inline applications/tari_dan_wallet_daemon/src/jrpc_server.rs]: remove

File: applications/tari_dan_wallet_web_ui/src/routes/AccountDetails/AccountDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                          </TableRow>
+                      </TableHead>
+                      <TableBody>
+                          <TableRow>
+                              <TableCell>{state?.account.name}
+                              </TableCell>
+                              <TableCell>{toHexString(state?.account.address.Component)}</TableCell>
+                             <TableCell>{state?.public_key}</TableCell>
+                          </TableRow>
+                              </TableBody>
```
Comment (Cifko) [inline applications/tari_dan_wallet_web_ui/src/routes/AccountDetails/AccountDetails.tsx]: nit pick: formatting is off, we should probably come up with some default formatting that we want to use for js/tsx files. I don't think that we have that.

File: applications/tari_dan_wallet_web_ui/src/routes/Wallet/Components/Accounts.tsx [hunk 4]
```diff
@@ -113,11 +118,12 @@ function Accounts() {
...
   }, []);
-  if (error) {
-    return <Error component="Accounts" message={error} />;
-  }
+
   return (
     <>
+      {error ? (
+          <Alert severity="error">{error}</Alert>
+      ) : ( <span></span>) }
```
Comment (Cifko) [inline applications/tari_dan_wallet_web_ui/src/routes/Wallet/Components/Accounts.tsx]: I think you can have `null` instead of empty `<span>` tag

**Review by Cifko — DISMISSED** [review]
utack

File: applications/tari_dan_wallet_daemon/src/handlers/accounts.rs (hunk 1 above)
Comment (stringhandler) [reply applications/tari_dan_wallet_daemon/src/handlers/accounts.rs]: lol thanks

File: applications/tari_dan_wallet_web_ui/src/routes/AccountDetails/AccountDetails.tsx (hunk 3 above)
Comment (stringhandler) [reply applications/tari_dan_wallet_web_ui/src/routes/AccountDetails/AccountDetails.tsx]: is there an autoformatter we can use and add to CI? eslint maybe?

File: applications/tari_dan_wallet_web_ui/src/routes/Wallet/Components/Accounts.tsx (hunk 4 above)
Comment (stringhandler) [reply applications/tari_dan_wallet_web_ui/src/routes/Wallet/Components/Accounts.tsx]: thanks

**Pull request merged**